sql/5_0_0-to-5_0_1_upgrade.sql

```diff
@@ -669,4 +669,8 @@ UPDATE `openemr_postcalendar_categories` SET pc_constant_id = LOWER(REPLACE (pc_
 
 #IfMissingColumn facility facility_taxonomy
 ALTER TABLE facility ADD facility_taxonomy VARCHAR(15) DEFAULT NULL;
-#EndIf
\ No newline at end of file
+#EndIf
+
+#IfTable form_therapy_groups_attendance
```

Comment:
Very nice to see your thinking about this!

Reply:
However, to make this bulletproof, place this above the code block you modified at line 258. This then means we won't be able to break it even if run the upgrade script multiple times.

Reply:
sure :+1: